client/src/main/java/com/netflix/conductor/client/automator/TaskPollExecutor.java

```diff
@@ -259,7 +254,7 @@ private void executeTask(Worker worker, Task task) {
                 worker.getClass().getSimpleName(),
                 worker.getIdentity(),
                 result.getStatus());
-        updateWithRetry(updateRetryCount, task, result, worker);
+        updateTaskResult(updateRetryCount, task, result, worker);
```

Comment:
updateRetryCount is a property that we expose to our client that can be configed. Our internal spring boot client library also expose that. Why do we want to remove the retries here?

Reply:
good catch, will need to add a retry loop here

Reply:
added a retry loop for update operations